spec/api-browser-window-spec.ts

```diff
@@ -1665,6 +1665,42 @@ describe('BrowserWindow module', () => {
           expectBoundsEqual(w.getNormalBounds(), bounds);
         });
 
+        it('correctly reports maximized state after maximizing then minimizing', async () => {
+          w.destroy();
+          w = new BrowserWindow({ show: false });
+
+          w.show();
```

Comment:
nit: is this a necessary part of the test, starting hidden and then showing?

Reply:
@dsanders11 strictly no, but it historically minimizes flakes in CI